docs/kyma/docs/032-gs-cluster-installation.md

```diff
@@ -0,0 +1,141 @@
+    "--authentication-token-webhook": "true",
+    "--authorization-mode": "Webhook"
+}
+```
+
+## Installation
+
+1. Create the `kyma-installer` Namespace.
+
+Run the following command:
+
+```
+kubectl create ns kyma-installer
+```
+
+2. Fill in the `installer-config.yaml.tpl` template.
+
+The Kyma installation process requires installation data specified in the `installer-config.yaml` file. Copy the `installer-config.yaml.tpl` template, rename it to `installer-config.yaml`, and fill in these placeholder values:
+
+- `__TLS_CERT__` for the TLS certificate
+- `__TLS_KEY__` for the TLS certificate key
+- `__REMOTE_ENV_CA__` for the Remote Environments CA
+- `__REMOTE_ENV_CA_KEY__` for the Remote Environments CA key
+- `__EXTERNAL_IP_ADDRESS__` for the IP address for Kyma Ingress
+- `__DOMAIN__` for the domain name, for example `kyma.example.com`
+- `__REMOTE_ENV_IP__` for the IP address for Remote Environments Ingress
+- `__K8S_APISERVER_URL__` for the API server's URL
+- `__K8S_APISERVER_CA__` for your API Server CA
+- `__ADMIN_GROUP__` for the additional admin group. This value is optional.
+- `__ENABLE_ETCD_BACKUP_OPERATOR__` for enabling or disabling the `etcd` backup operator. Enter `true` or `false`.
+- `__ETCD_BACKUP_ABS_CONTAINER_NAME__` for the Azure Blob Storage name for `etcd` backups. You can leave the value blank when the backup operator is disabled.
+
+**NOTE:** Because the `etcd` backup feature is in development, set `__ENABLE_ETCD_BACKUP_OPERATOR__` to `false` and leave `__ETCD_BACKUP_ABS_CONTAINER_NAME__` blank.
+
+When you fill in all required placeholder values, run the following command to provide the cluster with the installation data:
+
+```
+kubectl apply -f installer-config.yaml
+```
+
+3. Bind the default RBAC role.
+
+Kyma installation requires increased permissions granted by the **cluster-admin** role. To bind the role with the default **ServiceAccount**, run the following command:
+
+```
+kubectl apply -f resources/cluster-prerequisites/default-sa-rbac-role.yaml
+```
+
+4. Deploy `tiller`.
+
+To deploy the `tiller` component on your cluster, run the following command:
+
+```
+kubectl apply -f installation/resources/tiller.yaml
+```
+
+Wait for the `tiller` Pod to be ready. Execute the following command to check that it is running:
+
+```
+kubectl get pods -n kube-system | grep tiller
+```  
+
+5. Deploy the `Installer` component.
+
+To deploy the `Installer` component on your cluster, run this command:
+
+```
+kubectl apply -f installation/resources/installer.yaml -n kyma-installer
+```
+
+6. Trigger the installation.
+
+To trigger the installation of Kyma, you need a Custom Resource file. Copy the `installer-cr.yaml.tpl` file, rename it to `installer-cr.yaml`, and fill in these placeholder values:
+
+- `__VERSION__` for the version number of Kyma to install. When manually installing Kyma on a cluster, specify any valid [SemVer](https://semver.org/) notation string, for example `0.0.1`
```

Comment:
- `__VERSION__` for the version number of Kyma to install. When manually installing Kyma on a cluster, specify any valid [SemVer](https://semver.org/) notation string. For example, `0.0.1`